projects/bp-gallery/src/components/views/picture/sidebar/picture-info/DateRangeSelectionField.tsx

```diff
@@ -63,6 +63,10 @@ const DateRangeSelectionField = ({
     onTouch();
     setTimeRange(oldTimeRange => {
       const tRT = oldTimeRange ?? ({} as FlatTimeRangeTag);
+      if (!timeRange) {
+        tRT.start = Date.now();
+        tRT.end = Date.now();
```

Comment:
Calling `Date.now` twice in a row seems like something that could cause problems (if the second call returns a different timestamp). Could you either put the result in a variable or simply do `tRT.end = tRT.start`?

Reply:
having different values for start and end should not be a problem though, since the start value would always be before the end value, therefore making it a valid timerange, right?

Reply:
It would be valid, yes. But I think the intention with your code is to set both values to "now". So having two slightly different values (which might not even happen, not sure) is just annoying and if you see that in the database, surprising. That's my opinion anyway. One more reason to rewrite this would be the DRY principle.